### DISCUSSION TITLE
Error message

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#39] [General]
Now getting this:

![image](https://github.com/larspalo/GOODF/assets/6999990/1da1aafc-9051-4de5-8844-7cb1e7fb533c)

**Comment by larspalo** [comment]
That is a problem with the .png file. See https://stackoverflow.com/questions/22745076/libpng-warning-iccp-known-incorrect-srgb-profile and the same warning would (most likely) also appear in GO.

**Comment by Bumblebee001** [comment]
Indeed. I found how to change that in the software I am using. adobe 1998
setting accepted by GOODF.

**Comment by Bumblebee001** [comment]
![image](https://github.com/larspalo/GOODF/assets/6999990/c83270c4-bd42-4045-abe3-003a97552172)
:-)

**Reply by larspalo** [reply]
Great! I think you forgot to set TextBreakWidth to zero for the upper Hintersatz combination tab, but otherwise it looks very nice!

**Reply by Bumblebee001** [reply]
You have phenomenally good eyesight!!!!! God bless you!

**Reply by Bumblebee001** [reply]
(or mine getting really bad!)
:-D

**Reply by larspalo** [reply]
No problem! A new release is building right now. In about 40 minutes it should be ready but now I'm going to bed...

**Reply by Bumblebee001** [reply]
The Transparent element is confusing me somewhat. 
I did delete what appeared to me as reduntant elements..... (I know I did wrong.... somehow)
As things stand at the moment, I have no functioning H/ I/ II pistons.

**Comment by Bumblebee001** [comment]
Y E S S S S

**Comment by Bumblebee001** [comment]
![image](https://github.com/larspalo/GOODF/assets/6999990/34715e98-9ca7-47a9-8839-946602597ef7)

I need assistance for the remaining pistons and third Pedal tabs plus the H, I and II pistons.

**Reply by Bumblebee001** [reply]
PS I need to move everything down further to accomodate the third row of pedal stops.... but made great strides thanks to your GOODF.

**Reply by larspalo** [reply]
Yes, it seems that you need to adjust the layout somewhat. The first question is the size of the panel - could it be increased? Nowadays most monitors have a screen resolution of 1920x1080 which might not be fully usable with a 100% scaling as the window bar takes up some of the height. Recently I've settled for a main window size of (or around) 1600x900 which leaves some room for the activity bars of the OS and the possibility of having other panels open for easier access. Of course, by no means I suggest that you have to do things like I do, I just give you the recipe/approach I use right now.

I've more or less adopted a general scheme for panels that would include:
**Main Panel** (as close to original console as is resonably possible)
Then _in a subgroup_ for the sample set:
**Stops Panel** (no other things, just the controls in an easy to read fashion)
**Noise Controls Panel**
**Left Jamb**
**Right Jamb**
And _possibly additional panels_ for more decorative elements/purposes

This allows for a main panel that's not so cluttered with elements. In your case above I would immediately remove the Set button and the row of additional combinations that doesn't belong in the original organ from the main panel.

**Reply by Bumblebee001** [reply]
Thanks for your advice. This is not what I will be publishing. I am after achieving the functionality and sorting out issues. The free sample-set will be more or less the original but it has become my custom to add an extended set that attracts some donations. So the extended set will have more bells and whistles besides the modern 61+61+32 keyboards. 

Regarding the width of the screen.... yes I can afford to go more side-ways, not much though, but I have reached my y-axis limits. The GUI is currently far from ideal: text too small and stops and tabs can afford to be a bit larger. There would be a scope for left and right screens but how many organists out there have two touch screens.... one has to cater reasonably to cover most needs.

My current setup is like this (the laptop I use most of the time - NOT my home built organ):
![image](https://github.com/larspalo/GOODF/assets/6999990/1f6ea49d-8ae8-4a27-bfe5-7339037bf842)

**Comment by Bumblebee001** [comment]
Adjusted elements:

![image](https://github.com/larspalo/GOODF/assets/6999990/6daa2258-0047-4def-a0de-692f337a53ae)

**Reply by Bumblebee001** [reply]
The single pedal stop is just there to show where the third row would be and what the stops will look like. It is currently non functional other than coming on and off when the corresponding main (hand registration) tab is activated/ deactivated.... which it should not. I am just waiting for instructions on how to proceed.

![image](https://github.com/larspalo/GOODF/assets/6999990/078e7d3b-f8b4-4758-a290-4f3312e2329c)

**Reply by larspalo** [reply]
See the 5th point in https://github.com/larspalo/GOODF/discussions/42#discussioncomment-6549093

**Reply by Bumblebee001** [reply]
I think it is a strange and useless to have a piston to allow use of an extra row of stops. I need to clarify this with the local organ builder or re-visit the organ to see what the piston actually does. I might be doing so this coming Friday because I need to take a couple more photos too. Unfortunately I am not familiar with the organ myself and I know no one who regularly plays it. Recording the instrument and creating this sample-set were never on my mind and actually did so because my UK organist friend was interested in doing it whilst he was on holiday here and with me having failed to obtain permission to record another instrument elsewhere. Futhermore, this organ was outside the remit of my "Historic" project.

**Comment by Bumblebee001** [comment]
It seems like the order of entering switches can make all the difference. I cannot apply the Reeds Off (NOT) in most stops as I created it rather late in the day. Being where it is it does not appear in the list of switches for stops H, I, II.

**Reply by larspalo** [reply]
Yes, and that's why I've committed code that makes it possible to change the switch order by drag & drop in the tree. It's not trivial, and as switches only can reference other switches with a lower number the validation may remove switch references when re-ordering the switches, but that's only to be expected as they no longer are valid. But main point is that 4237a74d4c6ea4ada58b08e5d9b8bc7fa560ae6b will make it much easier to expand a switch network.
